api/jobs.go

```diff
@@ -471,6 +498,11 @@ func (j *Jobs) PlanOpts(job *Job, opts *PlanOptions, q *WriteOptions) (*JobPlanR
 	if opts != nil {
 		req.Diff = opts.Diff
 		req.PolicyOverride = opts.PolicyOverride
+		if opts.DiffVersion != nil {
+			req.DiffVersion = opts.DiffVersion
+		}
+
```

Comment:
These are both pointers, so this is safe and saves a comparison:

```suggestion
		req.DiffVersion = opts.DiffVersion
```